usaspending_api/accounts/views/financial_spending.py

```diff
@@ -24,7 +23,7 @@ def get_queryset(self):
 
         # required query parameters were not provided
         if not (fiscal_year and agency_id):
-            raise InvalidParameterException('Missing one or more required query parameters: fiscal_year, agency_id')
+            raise ParseError('Missing one or more required query parameters: fiscal_year, agency_id')
```

Comment:
the exception handler doesnt know how to handle ParseError, so it throws a 500 after.  It knows about InvalidParameterException though, so it can through a 400 after it gets raised. Thanks @nmonga91 for figuring this out.